docs/how-to/custom-base-templates.md

```diff
@@ -0,0 +1,161 @@
+# Use custom base HTML templates
+
+If the default template in the Starter Pack doesn't fully meet your needs -- whether you want a unique layout, a custom header or footer, or a specialized sidebar for certain pages -- you can create and use a custom base HTML template for your Sphinx project.
+
+This guide shows you how to extend or override the default templates to tailor the look and structure of your documentation.
+
+```{note}
+Base template customizations can be made to your documentation.
+However, they are not officially supported by the team maintaining the starter pack.
+Use them at your own discretion.
+```
+
+## Setup
+
+First, create the {file}`_templates` directory; all your custom templates will need to be stored in this folder.
+
+Then uncomment this line in {file}`conf.py` so your Sphinx project will use local templates:
+
+```{code-block} py
+:caption: conf\.py
+
+templates_path = ["_templates"]
+```
+
+In most cases, you will need to copy the templates for the [canonical-sphinx theme] as a starting point and edit as needed.
+
+## Use custom template for all pages
+
+Sphinx looks for a template called {file}`page.html` as the entry point and main page template for documentation pages.
+To customize your project's look and structure, check this file and determine which parts -- such as the header, footer, or sidebars -- need to be edited or overridden.
+
+Here are some examples.
+
+### Remove on-page TOC
+
+To remove the on-page TOC in the right sidebar, make a copy of [page.html] in the {file}`_templates` folder, and remove lines 2 - 14.
+This will apply to all pages. 
+
+```{code-block} html
+:emphasize-lines: 2-14
+:linenos:
+
+{% block right_sidebar %}
+<div class="toc-sticky toc-scroll">
+   {% if not furo_hide_toc_orig %}
+    <div class="toc-title-container">
+      <span class="toc-title">
+       {{ _("Contents") }}
+      </span>
+    </div>
+    <div class="toc-tree-container">
+      <div class="toc-tree">
+        {{ toc }}
+      </div>
+    </div>
+   {% endif %}
+    {% if meta and ((meta.discourse and discourse_prefix) or meta.relatedlinks) %}
+    <div class="relatedlinks-title-container">
+      <span class="relatedlinks-title">
+```
+
+### Add icon for GitHub link in header
+
+To customize the default header by adding an icon for the GitHub link, first make a copy of [header.html] in the {file}`_templates` folder.
+
+Then modify the conditional statement related to GitHub (see lines 5 - 13 below).
+
+```{code-block} html
+:linenos:
+:emphasize-lines: 5-13
+
+[...]     
```

Comment:
```suggestion
:emphasize-lines: 5-13

[...]     
```

The elided lines are in tension with the line numbers. I'd cut them.

We might also want to disable the copy button, since it'll preserve all the non-highlighted code.